pkgs/development/python-modules/pycoin/default.nix

```
@@ -0,0 +1,44 @@
+{ stdenv
+, fetchPypi
+, buildPythonPackage
+, setuptools
+, substituteAll
+, gnupg
+, pytestCheckHook }:
+
+buildPythonPackage rec {
+  pname = "pycoin";
+  version = "0.90.20200322";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "c8af579e86c118deb64d39e0d844d53a065cdd8227ddd632112e5667370b53a3";
+  };
+
+  propagatedBuildInputs = [
+    setuptools
+  ];
```

```suggestion
  propagatedBuildInputs = [ setuptools ];
```
Could optionally condense to 1 line, fine with me either way.